src/mumble_exe/CMakeLists.txt

```diff
@@ -45,3 +45,26 @@ endif()
 target_link_libraries(mumble_exe PRIVATE shlwapi.lib)
 
 install(TARGETS mumble_exe RUNTIME DESTINATION "${MUMBLE_INSTALL_EXECUTABLEDIR}" COMPONENT mumble_client)
+
+if(packaging)
+	set(overlay ON)
+	set(plugins ON)
+	
+	if(translations)
+		set(installer_vars "--all-languages")
+	endif()
+	
+	file(COPY 
+		${CMAKE_SOURCE_DIR}/installer/MumbleInstall.cs
+		${CMAKE_SOURCE_DIR}/installer/ClientInstaller.cs
+		DESTINATION
+			${CMAKE_BINARY_DIR}/installer/client
+	)
+	
+	add_custom_command(TARGET mumble_exe
+		POST_BUILD
+		COMMAND cscs.exe -cd MumbleInstall.cs 
```

Comment:
`cscs.exe` does not seem to be present on our Windows CI... I assume this has to be installed separately before building Mumble?

Reply:
Correct.

Reply:
What does this executable belong to? C# in general?
Do you already have an idea on how this could be installed using the command-line? :thinking:

Reply:
It's part of the WiX# bundle. @davidebeatrici  and I are in discussion about getting it added to the build environment.